README.md

```diff
@@ -43,14 +43,17 @@ The loading recommendation for each third-party resource is defined with the fol
     key?: string;
     url: string | {
       url: string;
-      params: Array<string> ;
     };
+    params: Array<string> ;
+    optionalParams?: Record<string, string | number | undefined | null>;
```

Comment:
`undefined` as well? Shouldn't `null` be sufficient?

Reply:
yeah in theory... we can remove it, but I rather wait until nuxt work is released as I don't want to introduce any distractions rn